src/app/pages/Home/Home.tsx

```diff
@@ -14,17 +14,26 @@ export default function Home(): JSX.Element {
     <div className={styles.container}>
       <Header></Header>
       <p className={styles.userInfo}>Your location:</p>
-      {cityIsLoading && data ? (
-        <p className={styles.loadingText}>Loading...</p>
-      ) : (
-        <LocationCard
-          locationName={cityName}
-          degree={data.degree}
-          icon={data.icon}
-          wind={data.wind}
-          description={data.description}
-        />
-      )}
+      {cityIsLoading && <p className={styles.loadingText}>Loading...</p>}
+      {cityName === '' &&
+        data.degree === 0 &&
+        data.icon === '' &&
+        data.wind === 0 &&
+        data.description === '' && <div></div>}
+      {cityIsLoading === false &&
+        cityName !== '' &&
+        data.degree !== 0 &&
+        data.icon !== '' &&
+        data.wind !== 0 &&
+        data.description !== '' && (
+          <LocationCard
+            locationName={cityName}
+            degree={data.degree}
+            icon={data.icon}
+            wind={data.wind}
+            description={data.description}
+          />
+        )}
```

Comment:
It works, but I think you should move this logic to a function or variable or something, so that it read more like
```suggestion
      {dataIsLoaded ? <LocationCard
            locationName={cityName}
            degree={data.degree}
            icon={data.icon}
            wind={data.wind}
            description={data.description}
          /> : <div></div>}
```
Or if that suggestion isn't working quite right, just something similar ;)